pom.xml

```diff
@@ -11,74 +11,181 @@
+                                    <id>default-cli</id>
+                                    <goals>
+                                        <goal>exec</goal>
+                                    </goals>
+                                    <configuration>
+                                        <arguments>
+                                            <argument>--module-path</argument>
+                                            <!-- automatically creates the modulepath using all project dependencies, also adding the project build directory -->
+                                            <modulepath/>
+                                            <argument>-classpath</argument>
+                                            <!-- automatically creates the classpath using all project dependencies, also adding the project build directory -->
+                                            <classpath/>
+                                            <argument>-XX:+UnlockExperimentalVMOptions</argument>
+                                            <argument>-XX:+EnableJVMCI</argument>
+                                            <argument>--upgrade-module-path=${compiler.dir}/compiler.jar</argument>
+                                            <argument>com.mycompany.app.App</argument>
+                                        </arguments>
+                                    </configuration>
+                                </execution>
+                                <execution>
+                                    <id>nograal</id>
+                                    <goals>
+                                        <goal>exec</goal>
+                                    </goals>
+                                    <configuration>
+                                        <arguments>
+                                            <argument>--module-path</argument>
+                                            <!-- automatically creates the modulepath using all project dependencies, also adding the project build directory -->
+                                            <modulepath/>
+                                            <argument>-classpath</argument>
+                                            <!-- automatically creates the classpath using all project dependencies, also adding the project build directory -->
+                                            <classpath/>
+                                            <argument>com.mycompany.app.App</argument>
+                                        </arguments>
+                                    </configuration>
+                                </execution>
+                            </executions>
+                            <configuration>
+                                <executable>${JAVA_HOME}/bin/java</executable>
+                            </configuration>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+                <plugins>
```

Comment:
If you remove `<pluginManagerment>`, then it will not be necessary to repeat the names of plugins in this section.